### ISSUE TITLE
Issue installing the Module at all versions. 

### ISSUE THREAD

**Issue by Hentik** [Skyl3lazer/giffyglyph-monster-maker-continued#12]
**Module:** Giffyglyph's 5e Monster Maker Continued

**Foundry and System version:** Foundry Version (V10 Build 303) , DND5e 2.2.2

**OS, Hosting, Browser (if applicable): Hosting :** Google Chrome, Win 10, Forge

**Short Description of bug:**

Module is not shown in Module selection screen.

**Comment by Skyl3lazer** [comment]
Could you paste the manifest link you're using to install?

**Comment by Skyl3lazer** [comment]
Also, are there any red errors in your console on the module screen?

**Comment by Hentik** [comment]
> Could you paste the manifest link you're using to install?

sure! here it is : 
https://raw.githubusercontent.com/Skyl3lazer/giffyglyph-monster-maker-continued/master/module.json

No red errors in the console related to the module. 
Thanks for the fast reply  :)

**Comment by Skyl3lazer** [comment]
Ah, I missed the version initially. I'm not yet fully compatible with dnd5e 2.2.2, it's in progress.

**Comment by Hentik** [comment]
Do you recommend to go back on the dnd 5e version? (would it work on 2.2.1?)

**Comment by Skyl3lazer** [comment]
> Do you recommend to go back on the dnd 5e version? (would it work on 2.2.1?)

You don't need to, I actually just tested and it looks like the module works correctly. I will note that on 10.303 there is an issue with some automation mods that will currently cause GGMMC monsters to break when effects are applied. These mods include midi/dae and dfred's convenient effects. 

I recommend using foundry v10.291 (or v11.304) if you experience those issues.

I've pushed those changes live so updating GGMMC should make it appear again correctly. Thank you for the report!
